redis/src/main/scala/zio/redis/api/SortedSets.scala

```diff
@@ -74,19 +75,19 @@ trait SortedSets {
 
   final val zScan = RedisCommand(
     "ZSCAN",
-    Tuple4(LongInput, OptionalInput(RegexInput), OptionalInput(LongInput), OptionalInput(StringInput)),
+    Tuple4(StringInput, LongInput, OptionalInput(RegexInput), OptionalInput(CountInput)),
     ScanOutput
   )
 
-  final val zScore = RedisCommand("ZSCORE", Tuple2(StringInput, StringInput), OptionalOutput(LongOutput))
+  final val zScore = RedisCommand("ZSCORE", Tuple2(StringInput, StringInput), OptionalOutput(StringOutput))
 
   final val zUnionStore = RedisCommand(
     "ZUNIONSTORE",
     Tuple4(
       StringInput,
-      NonEmptyList(StringInput),
-      OptionalInput(NonEmptyList(DoubleInput)),
-      OptionalInput(AggregateInput)
+      NonEmptyListPrecededByCount(StringInput),
+      OptionalInput(AggregateInput),
+      OptionalInput(WeightsInput)
```

Comment:
This isn't the default order as per the docs.

Reply:
hmm will check it